deviantart/deviantart_scratcher.py

```diff
@@ -16,26 +15,32 @@
 from requests.adapters import HTTPAdapter
 from urllib3.util.retry import Retry
 
-sys.path.append(".")
 # First-party/Local
-import quantify  # noqa: E402
+import quantify
 
-PATH_REPO_ROOT, PATH_WORK_DIR, PATH_DOTENV, DATETIME_TODAY = quantify.setup(
-    __file__
+# Setup paths, Date and LOGGER using quantify.setup()
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
+PATH_REPO_ROOT, PATH_WORK_DIR, PATH_DOTENV, DATETIME_TODAY, LOGGER = (
+    quantify.setup(__file__)
 )
+
+# Load environment variables
 load_dotenv(PATH_DOTENV)
 
-# Retrieve API keys
-API_KEYS = os.getenv("GOOGLE_API_KEYS").split(",")
+
+# Global Variable for API_KEYS indexing
 API_KEYS_IND = 0
+
+# Gets API_KEYS and PSE_KEY from .env file
+API_KEYS = os.getenv("GOOGLE_API_KEYS").split(",")
+PSE_KEY = os.getenv("PSE_KEY")
+
 # Set up file path for CSV report
-DATA_WRITE_FILE = os.path.join(
-    PATH_WORK_DIR,
-    f"data_deviantart_"
-    f"{DATETIME_TODAY.year}_{DATETIME_TODAY.month}_{DATETIME_TODAY.day}.csv",
+DATA_WRITE_FILE = (
+    f"{PATH_WORK_DIR}"
```

Comment:
But we are getting/finding the `PATH_WORK_DIR` from the shared module of `quantify.py`, and if we again use the `os.path.join()` here, wouldn't it will be a redundancy to again get the path for the working directory.